conceptnet5/vectors/transforms.py

```diff
@@ -69,8 +69,12 @@ def l2_normalize_rows(frame):
     L_2-normalize the rows of this DataFrame, so their lengths in Euclidean
     distance are all 1. This enables cosine similarities to be computed as
     dot-products between these rows.
-    DataFrame of zeros will be normalized to zeros.
+
+    Rows of zeroes will be normalized to zeroes, and frames with no rows will
+    be returned as-is.
     """
+    if frame.shape[0] == 0:
```

Comment:
Even though I don't think there is a way in our code to get an empty frame and then call `l1_normalize_columns` on it, should we add this check there as well?

Reply:
If we end up trying to normalize 0 columns, that seems to me like something actually has gone wrong.